identifiers/country-ar/README.md

```diff
@@ -0,0 +1,9 @@
+# OCD Division Identifier
+
+This folder list the following administrative divisions of Argentina : 
+* Provinces of Argentina
+  [ref](https://en.wikipedia.org/wiki/Provinces_of_Argentina) as **state**.
```

Comment:
Since these are referred to as provinces, I think it would make more sense to use `province` instead of `state` like we do for Canada.

Reply:
I agree. File updated.